slatedb/src/mem_table.rs

```diff
@@ -233,6 +248,7 @@ impl KVTable {
             durable: WatchableOnceCell::new(),
             last_tick: AtomicI64::new(i64::MIN),
             last_seq: AtomicU64::new(0),
+            sequence_tracker: Mutex::new(SequenceTracker::new()),
```

Comment:
rather than using the default capacity/interval, how about setting the capacity and interval to `u64:::MAX` and `0`, respectively to have these trackers just track everything. I think with the current config we'd wind up potentially dropping some sequence numbers in the pathological case where you have a memtable that filled up really slowly with lots of small writes. You'd also wind up preserving a different set of seq numbers than the old algorithm.

You'd also need to make sure to not create the `Vec`s holding the timestamps/seqnums with capacity `u64::MAX`